starport-core/src/main/scala/com/krux/starport/db/tool/SubmitPipeline.scala

```diff
@@ -55,7 +55,7 @@ object SubmitPipeline extends DateTimeFunctions with WaitForIt with DateTimeMapp
 
     def runQuery[T](query: DBIO[T], dryRunOutput: T, force: Boolean = false): T = {
       if (opts.dryRun && !force) {
-        println("Dry Run. Skip sending the querys...")
+        println("dry run...skipping queries")
```

Comment:
there should be a space after `...` right?

Reply:
Not usually, no.